.github/workflows/ci.yml

```diff
@@ -0,0 +1,28 @@
+name: Unit tests, QA
+
+on:
+  push:
+    branches: [ '**' ]
```

Comment:
Also maybe you can use common YAML syntax for lists. Or do you prefer the abbreviated way? https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html
What do you think of the YAML syntax? 
@mariajgrimaldi @JuanDavidBuitrago 
e.g.
```
branches:
  - main
  - '**'
```

Reply:
You're right @johanv26 !! Is better use an array list.